core/src/main/java/bio/terra/pearl/core/service/kit/KitRequestService.java

```diff
@@ -307,6 +315,30 @@ public KitRequest assemble(
         return kitRequest;
     }
 
+    public KitRequest insertKitRequest(
```

Comment:
insert a kit_request entry form passed kitRequestDto

Reply:
as mentioned above, this logic should be in the EnrolleeImportService, where it can be accompanied by the time-shifting code if needed